src/client.ts

```diff
@@ -29,4 +41,43 @@ export class EdcConnectorClient {
   static version(): string {
     return version;
   }
+
+}
+
+export class EdcConnectorClientBuilder {
```

Comment:
I did a quick investigation and full privatization of fields should be doable in plain JS.

Consider the following example.
```js
class A {
  #hidden = undefined;

  log() {
    console.log(this.#hidden);
  }

  static Builder = class Builder {
    #instance = new A();

    hidden(hidden) {
      this.#instance.#hidden = hidden;
      return this;
    }

    build() {
      return this.#instance;
    }
  };
}
```

After copy/paste the above in the browser console you can see its functioning
```js
new A.Builder()
  .hidden("ciao")
  .build()
  .log() // "ciao"
```

So, after all we should be able to use native JS private fields without `Symbol` nor TypeScript's `private` keyword